src/dos/cdrom.cpp

```diff
@@ -28,6 +28,10 @@
 #include "pic.h"
 #include "string_utils.h"
 
+namespace CDROM {
+std::array<std::unique_ptr<CDROM_Interface>, 26> cdroms;
+}
```

Comment:
Do we even need this namespace? If `cdroms` is module-internal, we don't need it, then if external modules want access the CDROM_Interface, probably a `CDROM_GetDrive()` or something would be better?

Reply:
Yeah, it makes sense then and it's ok for now.